Data-Structures/src/test/java/linkedlist/LinkedListTest.java

```diff
@@ -0,0 +1,107 @@
+package linkedlist;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+
+    @Test public void linkedList_testEmptyList() {
+
+        LinkedList<Integer> emptyIntegerList = new LinkedList<>();
+
+        int expected = 0;
+        int actual = emptyIntegerList.getSize();
+
+        assertEquals(expected, actual);
+    }
+
```

Comment:
If there is setup you're repeating for each test you can use @Before to run before each test. For example;

LinkedList<Integer> intLinkedList = new LinkedList<>();
@Before 
public void setup(){
        intLinkedList.insert(22);
        intLinkedList.insert(8);
        intLinkedList.insert(54);
        intLinkedList.insert(7);
}

Will setup the linked list before each test.